cli/tests/pcluster/cli/configure/test_pcluster_configure.py

```diff
@@ -689,14 +692,64 @@ def general_wrapper_for_prompt_testing(
 ):
     MockHandler(mocker)
     input_composer = ComposeInput(aws_region_name=region, key=key, scheduler=scheduler)
-    input_composer.add_first_flow(op_sys, max_size, head_node_instance, compute_instance)
+    input_composer.add_first_flow(op_sys, head_node_instance)
+    input_composer.add_compute_instance(compute_instance, max_size)
     input_composer.add_no_automation_no_empty_vpc(vpc_id, head_node_id, compute_id)
     input_composer.mock_input(mocker)
 
     _run_configuration(mocker, temp_path_for_config)
     return True
 
 
+@pytest.mark.parametrize(
+    "test_dir_name, supports_efa, efa_enabled, placement_group_names, placement_group_exists",
+    [
+        ("test_enabled_efa_default_placement_group", True, "y", [""], lambda x: False),
```

Comment:
what about a test with supports_efa = False?

Reply:
Most of the other tests in `test_pcluster_configure` cover such a scenario, but I can explicitly include it here to be sure.